.github/workflows/auto-approve.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-kafka-schemas@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-protos@') ||
         startsWith(github.event.issue.title, 'publish: getsentry/sentry-redis-tools@') ||
+        startsWith(github.event.issue.title, 'publish: getsentry/service-registry@') ||
```

Comment:
repo doesn't have any CI validation included in branch protection

Reply:
@asottile-sentry Added status check rule: https://github.com/getsentry/service-registry/settings/branch_protection_rules/54806613

Reply:
<img width="495" alt="Screenshot 2024-10-01 at 4 20 09 PM" src="https://github.com/user-attachments/assets/ff73a50c-42b5-4ee8-a9ff-b7e1434b5c0b">

hmm I'm not seeing them

Reply:
<img width="769" alt="Screenshot 2024-10-01 at 1 33 47 PM" src="https://github.com/user-attachments/assets/6cf9ca8b-c94d-41d8-8de7-227d3099c753">
@asottile-sentry figured it out, the UI was just difficult to use & didn't list them